### PR TITLE
Iss04

### PR DESCRIPTION
**Pull request by yangleicq**
Address bioconductor review comments

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -4,31 +4,34 @@ Title: Count model based differential expression and normalization on GeoMx RNA
...
+Description: A series of statistical models using count generating distributions for background modelling, feature and sample QC, normalization and differential expression analysis on GeoMx RNA data. 
+            The application of these methods are demonstrated by example data analysis vignette.
 Imports:
     magrittr,
     Matrix,
     robust,
     plyr,
     lme4,
     Rcpp (>= 1.0.4.6), 
-    NanoStringNCTools,
```
Comment (zhiiiyang) [inline DESCRIPTION:15]: This line of change will cause an error in `devtools::check()`. 

```
> checking package dependencies ... ERROR
  Namespace dependency missing from DESCRIPTION Imports/Depends entries: 'NanoStringNCTools'
  
  See section 'The DESCRIPTION file' in the 'Writing R Extensions'
  manual.
```

**Review by zhiiiyang — CHANGES_REQUESTED** [review]
In addition, could you run `devtools::document()` to re-generate the man files?

File: DESCRIPTION [hunk 2]
```diff
@@ -4,31 +4,34 @@ Title: Count model based differential expression and normalization on GeoMx RNA
 Version: 0.99.3
 Authors@R: c(person("Lei", "Yang", email = "lyang@nanostring.com", role = c("aut")), 
              person("Zhi", "Yang", email = "zyang@nanostring.com", role = c("cre", "aut")))
-Description: We developed a series of statistical models using count generating distributions for fundamental tasks including background modelling, target and sample QC, normalization and differential expression analysis. The new methods are shown to be more suitable for these tasks in real examples than some conventional methods. We present GeoDiff, an R package for count generating models for analyzing Geomx RNA data, demonstrated by example data analysis pipeline vignette.
+Description: A series of statistical models using count generating distributions for background modelling, feature and sample QC, normalization and differential expression analysis on GeoMx RNA data. 
+            The application of these methods are demonstrated by example data analysis vignette.
```
[the remark this answers is not in the document]
Comment (yangleicq) [reply DESCRIPTION]: done

**Comment by zhiiiyang** [comment]
looks good!

**Pull request merged**